xbmc/guilib/Geometry.h

```diff
@@ -273,8 +273,54 @@ template <typename T> class CRectGen
   }
 };
 
+template <typename T> struct SizeGen
```

Comment:
why don't you move it to some windows only file? I think this is much better than those kone of workarounds: https://github.com/xbmc/xbmc/pull/12377/commits/17a9442bda35881ef11224bb2c8b3fad874ffee8